nml/actions/action2var_variables.py

```diff
@@ -699,7 +710,10 @@ def industry_cargotype(name, args, pos, info):
     'town_zone'             : {'var': 0x44, 'start': 0, 'size':  8},
     'random_bits'           : {'var': 0x5F, 'start': 8, 'size':  2},
 }
-# Railtypes have no 60+x variables
+
+varact2vars60x_railtype = {
```

Comment:
Most of the variables are missing from NFO docs. This one is also missing from OTTD implementation at least for rail/road/tram types.